schedule/schedule-appengine/src/main/i18n/locale-en.json

```diff
@@ -213,5 +215,17 @@
   "VALUE_MUST_BE_LESS_THAN": "Value must be less than {{value}}",
   "VIEW_SUBSCRIBERS": "View Subscribers",
   "WEDNESDAY_ABBREVIATED": "Wed",
-  "WIDGETS": "Widgets"
+  "WIDGETS": "Widgets",
+  "HISTORY_TYPE": "History event type",
+  "HT_ALL": "Show all events",
```

Comment:
Yes. These translations are already in use under #/admin/histories
On Di., 18. Aug. 2015 at 23:31 wszarmach <notifications@github.com> wrote:

> In schedule/schedule-appengine/src/main/i18n/locale-en.json
> <https://github.com/krico/jas/pull/644#discussion_r37356208>:
>
> > @@ -213,5 +215,17 @@
> >    "VALUE_MUST_BE_LESS_THAN": "Value must be less than {{value}}",
> >    "VIEW_SUBSCRIBERS": "View Subscribers",
> >    "WEDNESDAY_ABBREVIATED": "Wed",
> > -  "WIDGETS": "Widgets"
> > +  "WIDGETS": "Widgets",
> > +  "HISTORY_TYPE": "History event type",
> > +  "HT_ALL": "Show all events",
>
> Is the intention of adding the translations so that clients can eventually
> also see the events?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/krico/jas/pull/644/files#r37356208>.
>